asset/expander.go

```diff
@@ -90,3 +91,13 @@ func sniffType(f io.ReadSeeker) (filetype_types.Type, error) {
 
 	return ft, nil
 }
+
+// cleanup of the assetSHA when cache dir gets force deleted
+func CleanUp(fullPath string) error {
+	errorSHA := os.RemoveAll(fullPath)
+	if errorSHA != nil {
+		return errorSHA
+	}
+	return nil
+
+}
```

Comment:
This is equivalent to os.RemoveAll and its use can be replaced with os.RemoveAll.

Reply:
This change has been made